app/components/Views/confirmations/components/Approval/TemplateConfirmation/Templates/index.ts

```
@@ -14,6 +14,7 @@ export interface ConfirmationTemplateValues {
   onConfirm?: (opts?: AcceptOptions) => void;
   onlyConfirmButton?: boolean;
```

Since this does the same for cancel, is it worth renaming this to `hideCancelButton` for consistency?

Meaning both are visible by default but you can hide each one with an optional property.

We don't have many templates in mobile so should be easy to update any references.

---

Renamed in https://github.com/MetaMask/metamask-mobile/pull/8844/commits/b5fb540e43619ba9b1e24b511ede710f0d76f706